### PR TITLE
Download only new hash files

### PR DESCRIPTION
**Pull request by ktneely**
Modified the download script so it only downloads newly added hash files from virusshare, in order to conserve bandwidth for both sides.

### CODE REVIEW

**Comment by 3c7** [comment]
Hey @ktneely, can you base your PR on the development branch? That would be cool. Thanks for your contribution.

**Comment by nadouani** [comment]
@3c7 I've rebased the PR on `develop`

File: analyzers/Virusshare/getHashes.sh [hunk 1]
```diff
@@ -21,9 +21,16 @@ if [ ! -d $1 ]; then
 fi
 
 cd $1
-for u in `curl https://virusshare.com/hashes.4n6 | grep -E "hashes\/VirusShare_[0-9]{5}\.md5" | cut -d\" -f2`
+for u in `curl https://virusshare.com/hashes.4n6 | grep -E "VirusShare_[0-9]{5}\.md5" | c\
+ut -d\" -f2 | cut -d\/ -f2`
 do
     echo $u
-    wget https://virusshare.com/$u
+    if [ -e $1/$u ]; then
```
[the remark this answers is not in the document]
Comment (ktneely) [reply analyzers/Virusshare/getHashes.sh]: The script does `cd` into `$1/`, but when I'm checking the file names, the update performs two `cut` commands, lopping off the directory assigned to `$1`.  A little convoluted, but it makes the comparison easier.

[the remark this answers is not in the document]
Comment (3c7) [reply analyzers/Virusshare/getHashes.sh]: That's not what I wanted to say. You `cut`something off, that's right but it only affects the filename, after that you still have that commands:
- `cd $1`
- `if [ -e $1/$u ]`

That leads to "being in" directory `$1` and checks if there's an directory `$1` with file `$u` in it. Or I'm completely blind and oversee something. :D

[the remark this answers is not in the document]
Comment (ktneely) [reply analyzers/Virusshare/getHashes.sh]: prior to the `cd`, the script `cut`s the full path twice, making `u` only the file name as pulled from the virusshare website, which it then compares to the filenames in directory `$1`.

See, e.g. `| cut -d\" -f2 | cut -d\/ -f2`  in the update as opposed to only `cut -d\" -f2` in the original.  

The first cuts down to e.g. "hashes/VirusShare_00049.md5", the second lops off the directory on the hosting server and leaves only e.g. "VirusShare_00049.md5", which is then compared to the filename in `$1`.

**Pull request merged**